argparse.go

```diff
@@ -4,17 +4,25 @@ package argparse
 import (
 	"errors"
 	"fmt"
+	"io/ioutil"
+	"log"
 	"os"
 	"strings"
 )
 
+var (
+	//logger = log.New(os.Stderr, "logger: ", log.Ltime|log.Lshortfile)
+	logger = log.New(ioutil.Discard, "logger: ", log.Ltime|log.Lshortfile)
+)
+
```

Comment:
I don't think this package requires import of `ioutil` and `log`, not sure why this would even need to be here as package does not output anything to anywhere. It appears that you are trying to use logging for debugging, I don't think this is a good approach, may I suggest you look at delve (or better a proper IDE that will help with debugging the code, such as GoLand as the one I am using).

Reply:
Agree, I was going to remove them anyway.